atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/client/creation/DefaultCqlClientFactory.java

```diff
@@ -31,7 +31,7 @@
 import java.util.Set;
 import java.util.function.Supplier;
 
-public class DefaultCqlClientFactory implements CqlClientFactory {
+public final class DefaultCqlClientFactory implements CqlClientFactory {
```

Comment:
I thought this was like the default check we ship with baseline?

Reply:
Oh, you mean the "design for extension" check. Looks like we have that suppressed!